include/RAJA/Layout.hxx

```diff
@@ -55,9 +55,9 @@
 
 #include <iostream>
 #include <limits>
-#include "RAJA/IndexValue.hxx"
-#include "RAJA/LegacyCompatibility.hxx"
-#include "RAJA/foralln/Permutations.hxx"
+#include "RAJA/internal/IndexValue.hxx"
+#include "RAJA/internal/LegacyCompatibility.hxx"
+#include "RAJA/internal/foralln/Permutations.hxx"
```

Comment:
Actually, a package diagram would be helpful. If I have time, I'll try to throw one together tomorrow or Friday. This would make it easier to see what should go where.